crates/c_api/src/engine.rs

```diff
@@ -19,6 +19,7 @@ wasmi_c_api_macros::declare_own!(wasm_engine_t);
 ///
 /// Wraps [`wasmi::Engine::default`].
 #[no_mangle]
+#[cfg_attr(feature = "prefix-symbols", export_name = "wasmi_wasm_engine_new")]
```

Comment:
I am currently thinking about a solution to prevent having to duplicate the names of all those numerous functions for this `cfg_attr`. Technically we could craft a proc. macro attribute that can do this automatically, however, the big downside is that this proc. macro would require a dependency on `syn` which is huge. 🤔